app/backend/gwells/settings/__init__.py

```diff
@@ -65,6 +65,10 @@
 # Fixtures dirs
 FIXTURES_DIRS = [FIXTURES_DIR]
 
+# GeoDjango External Library Paths
+GDAL_LIBRARY_PATH = get_env_variable('GDAL_LIBRARY_PATH', '/usr/local/lib/libgdal.so')
```

Comment:
Just a note - this causes all kinds of trouble the moment you try to run local. Once you introduce these variables, I think it pretty much forces you to only work in docker.